README.md

```diff
@@ -29,237 +29,48 @@ Have a look at a real-world application of Mara Schema in the [Mara Example Proj
  
 &nbsp;
 
-## Defining entities, attributes, metrics & data sets
```

Comment:
Personally, I really appreciate readmes with an overview/example to see what the package can do. But links to the docs are the next best thing...

Reply:
You are right. I just couldn't help but did not want to have two places where the documentation is maintained. Do you have a better idea?

Reply:
I put the context now back again and fixed links for PyPI when using setup.cfg